sdv/metadata/multi_table.py

```diff
@@ -532,6 +533,79 @@ def validate(self):
                 'The metadata is not valid' + '\n'.join(str(e) for e in errors)
             )
 
+    def _validate_missing_tables(self, data):
+        errors = []
+        missing_tables = set(self.tables) - set(data)
+        if missing_tables:
+            errors.append(f'The provided data is missing the tables {missing_tables}.')
+
+        return errors
+
+    def _validate_all_tables(self, data, table_synthesizers=None):
```

Comment:
It might be helpful to add a docstring here explaining what this validation method checks for. Either here or in the docstring for the `validate_data` method. Same with `_validate_foreign_keys`.